packages/botonic-core/src/utils.js

```diff
@@ -1,13 +1,11 @@
-export const isBrowser = () => {
-  return typeof window !== 'undefined' && !window.process
-}
+export const isBrowser =
```

Comment:
this is an API break. Please flag as "to document" and add doc for Anne.
Also, we shouldn't break compatibility in a patch upgrade.
delay until 0.17 or split into 2 PRs

Reply:
it is not breaking current functioning with webpack 4

Reply:
what I mean is that you changed from a function to a constant, so the bots which used it will need to drop the `()`

Reply:
okay, now I got it, so should we do it later this change? I can revert the commit